packages/drivers/odsp-driver/src/tokenFetch.ts

```diff
@@ -44,6 +44,12 @@ export interface TokenFetchOptions {
 export interface OdspResourceTokenFetchOptions extends TokenFetchOptions {
     /** Site url representing ODSP resource location */
     siteUrl: string;
+
+    /** ODSP drive id where resource resides. Optional, used only when fetching token to access ODSP file */
+    driveId?: string;
```

Comment:
Should it be required property?

Reply:
I think we need to retain ability to ask for a token without specifying file identifiers. This may be needed for other odsp-driver functionality such as discovering root drive id or default site for organization. I already see some code in FF that uses odsp-driver for that.